ccw.core/src/java/ccw/core/StaticStrings.java

```diff
@@ -0,0 +1,9 @@
+package ccw.core;
+
+public class StaticStrings {
```

Comment:
I see, I'm still not sure I like having a big file with all constants versus have the constants closer to where they naturally belong ... but at least the file is very clear, each section is well documented, so that's not a big deal right now